autogen.sh

```diff
@@ -130,6 +133,15 @@ case $do_autoconf,$do_git in
     do_autoconf=true;;
 esac
 
+if $require_nightly; then
+    echo "Validation Rust install ..."
+    command -v rustc >/dev/null 2>&1 || { echo >&2 "Remacs requires rust to be installed in order to build. Please install it via rustup: https://www.rustup.rs/; Aborting."; exit 1; }
+    command -v cargo >/dev/null 2>&1 || { echo >&2 "I require foo but it's not installed.  Aborting.Remacs requires cargo to be installed in order to build. Please install it via rustup: https://www.rustup.rs/; Aborting."; exit 1; }
+
+    rustc --version | grep nightly || { echo >&2 "Remacs currently requires nightly Rust. If you do not have nightly Rust, you should install rustup at https://www.rustup.rs/, and run 'rustup install nightly && rustup default nightly"; exit 1; }
```

Comment:
I'd suggest `rustup override ...` instead, to make the nightly choice only for Remacs.